jenkins/job.sample.yaml

```diff
@@ -31,6 +35,7 @@ hosts:
     on-package-updates:
     - apt:
         autoremove: '{{ autoremove }}'
+        withnewpkgs: '{{ withnewpkgs }}'
```

Comment:
`with-new-pkgs` here as well, but keep the parameter name

Reply:
done